src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -666,6 +667,7 @@ protected override void OnPointerReleased(PointerRoutedEventArgs args)
 		protected override void OnKeyDown(KeyRoutedEventArgs args)
 		{
 			base.OnKeyDown(args);
+			((IHandleableRoutedEventArgs)args).ShouldPreventDefaultIfHandled = false;
```

Comment:
```suggestion
		bool IHandleableRoutedEventArgs.DoNotPreventDefaultIfHandled { get; set; }
```

Reply:
Looks like this suggestion is a copy/paste error?

Should be:

```suggestion
			((IHandleableRoutedEventArgs)args).DoNotPreventDefaultIfHandled = true;
```

Reply:
Hey hey yes your right :)